css-layout-classes/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CSS Grid Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container margin75">
+    <div class="header-row"> <!-- first row which includes the header -->
+      <h1>Choose your character!</h1>
+    </div> <!-- end of first row -->
+    <div class="content-row"> <!--Second row of content-->
```

Comment:
Why do you have differently named `row` classes? You should only have one `row` class that you reuse.

Reply:
Sounds good, I have made the adjustment and have pushed the changes.